hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -1318,6 +1319,7 @@ public TableBuilder fromProperties(Properties properties) {
       }
       if (hoodieConfig.contains(HoodieTableConfig.TIMELINE_TIMEZONE)) {
         setCommitTimezone(HoodieTimelineTimeZone.valueOf(hoodieConfig.getStringOrDefault(HoodieTableConfig.TIMELINE_TIMEZONE)));
+        HoodieInstantTimeGenerator.setCommitTimeZone(HoodieTimelineTimeZone.valueOf(hoodieConfig.getStringOrDefault(HoodieTableConfig.TIMELINE_TIMEZONE)));
```

Comment:
we can remove this lline. its repeated

Reply:
removed and also ensure the default timezone is set while creating table config:
<img width="887" alt="Screenshot 2025-03-28 at 12 01 39 AM" src="https://github.com/user-attachments/assets/f03b4d5a-730d-44ae-b8da-25c7e97e6f82" />